docs/001_develop/03_client-capabilities/008_interaction/dialog.mdx

```diff
@@ -0,0 +1,298 @@
+          #dialogRef
+          id="example1"
+          class="example-dialog"
+          aria-label="Simple modal dialog"
+          modal="true"
+          hidden
+        >
+          <h2>Dialog</h2>
+          <p>This is an example dialog.</p>
+        </rapid-dialog>
+      `,
+      standalone: true,
+      schemas: [CUSTOM_ELEMENTS_SCHEMA],
+      imports: [FormsModule],
+    })
+    export class MyComponent {
+      @ViewChild('dialogRef') dialogRef!: ElementRef;
+
+      openDialog() {
+        this.dialogRef.nativeElement?.removeAttribute('hidden');
+        this.dialogRef.nativeElement?.show();
+      }
+    }
+    ```
+  </TabItem>
+</Tabs>
+
+
+## API
+
+### Attributes
+
+<table>
+    <thead>
+        <tr>
+            <th>Attribute</th>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Example</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>modal</td>
+            <td><code>boolean</code></td>
+            <td>Indicates the element is modal. When modal, user mouse interaction will be limited to the contents of the element by a modal overlay. Clicks on the overlay will cause the dialog to emit a "dismiss" event. Default value is true.</td>
+<td>
+
+```typescript
+<rapid-dialog modal>
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>hidden</td>
+            <td><code>boolean</code></td>
+            <td>The hidden state of the element. Default value is false.</td>
+<td>
+
+```typescript
+<rapid-dialog hidden>
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>trap-focus</td>
+            <td><code>boolean</code></td>
+            <td>Indicates that the dialog should trap focus. Default value is true.</td>
+<td>
+
+```typescript
+<rapid-dialog trap-focus>
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>aria-describedby</td>
+            <td><code>string</code></td>
+            <td>The id of the element describing the dialog.</td>
+<td>
+
+```typescript
+<rapid-dialog aria-describedby="description-id">
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>aria-labelledby</td>
+            <td><code>string</code></td>
+            <td>The id of the element labeling the dialog.</td>
+<td>
+
+```typescript
+<rapid-dialog aria-labelledby="label-id">
+```
+
+</td>
+        </tr>
+        <tr>
+            <td>aria-label</td>
+            <td><code>string</code></td>
+            <td>The label surfaced to assistive technologies.</td>
+<td>
+
+```typescript
+<rapid-dialog aria-label="Dialog title">
+```
+
+</td>
+        </tr>
+    </tbody>
+</table>
+
+### Properties
+
+This component doesn't have any properties which are not also controlled via attributes.
+
+### Methods
+
+|Method|Description|Example|
+|--|--|--|
+|show|Show the dialog|`dialogRef.show()`|
+|hide|Hides the dialog|`dialogRef.hide()`|
+
+### Slots
+
+|Slot|Description|
+|---|---|
+|Default|All the content which you want to display on the dialog box when shown.|
```

Comment:
I think some slots are missing, looking in foundation-ui dialog.ts there are slots like "top".